modules/appeals_api/app/sidekiq/appeals_api/evidence_submission_backup.rb

```diff
@@ -38,10 +38,10 @@ def notify(retry_params)
 
     def preloaded_evidence_submissions
       evidence_submissions = EvidenceSubmission.uploaded.preload(:supportable)
-
-      preloader = ActiveRecord::Associations::Preloader.new
-      preloader.preload(evidence_submissions.select { |es| APPEAL_TYPES.include?(es.supportable_type) },
-                        supportable: :status)
+      supported_evidence_submissions = evidence_submissions.select { |es| APPEAL_TYPES.include?(es.supportable_type) }
+      preloader = ActiveRecord::Associations::Preloader.new(records: supported_evidence_submissions,
+                                                            associations: { supportable: :status })
+      preloader.call
```

Comment:
The is the new way to preload data in Rails 7. Upon upgrading we were getting an argument error (`ArgumentError: missing keywords: :records, :associations`)